src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -334,8 +336,68 @@ describe('check Suite event handler', (): void => {
         },
       });
 
-    await checkSuiteHandle(octokit, 'dependabot-preview[bot]');
+    await checkSuiteHandle(octokit, 'dependabot-preview[bot]', 3);
 
     expect(warningSpy).toHaveBeenCalled();
   });
+
+  it('retries up to two times before failing', async (): Promise<void> => {
+    expect.assertions(6);
+
+    nock('https://api.github.com')
+      .post('/graphql')
+      .reply(OK, {
+        data: {
```

Comment:
Can you put the contents of this in a generator?

Reply:
I can, but I feel your request should go into a different PR. Every test has a mocked endpoint and their `data` content can be moved into a generator. I would rather prefer to have a dedicated PR to it.

Reply:
Ok sure, it's just to dry things out as I saw it was quite verbose and duplicated.